tests/TestCase.php

```diff
@@ -49,26 +50,10 @@ protected function getPackageProviders($app)
             NovaServiceProvider::class,
             NovaApplicationServiceProvider::class,
             FroalaFieldServiceProvider::class,
+            TestServiceProvider::class,
         ];
     }
 
-    /**
-     * Define environment setup.
-     *
-     * @param  \Illuminate\Foundation\Application  $app
-     * @return void
-     */
-    protected function getEnvironmentSetUp($app)
-    {
```

Comment:
Why you have removed default DB connection setup?

Reply:
See https://github.com/froala/nova-froala-field/pull/77#discussion_r634469691